nautobot/core/settings.py

```diff
@@ -688,9 +688,6 @@
 # A value of None or 0 means results will never expire (depending on backend specifications).
 CELERY_RESULT_EXPIRES = None
 
-# If set to True, result messages will be persistent. This means the messages won’t be lost after a broker restart.
-CELERY_RESULT_PERSISTENT = True
```

Comment:
This only applies to the [rpc backend](https://docs.celeryq.dev/en/stable/userguide/configuration.html#std-setting-result_persistent) which Nautobot does not use